expressions_api/expression_verify.go

```diff
@@ -29,13 +29,13 @@ type ExpressionVerificationAccessorImpl struct {
 func NewExpressionVerificationAccessorImpl(ctx context.Context, project string, instance string) (*ExpressionVerificationAccessorImpl, error) {
 	var spannerAccessor *spanneraccessor.SpannerAccessorImpl
 	var err error
-	if project != "" && instance != "" {
-		spannerAccessor, err = spanneraccessor.NewSpannerAccessorClientImplWithSpannerClient(ctx, fmt.Sprintf("projects/%s/instances/%s/databases/%s", project, instance, constants.TEMP_DB))
+	if project == "" || instance == "" {
+		spannerAccessor, err = spanneraccessor.NewSpannerAccessorClientImpl(ctx)
 		if err != nil {
 			return nil, err
 		}
 	} else {
-		spannerAccessor, err = spanneraccessor.NewSpannerAccessorClientImpl(ctx)
```

Comment:
[Super-nit, can be a todo]
Could the Spanner resource URL generation `projects/%s/instances/%s/databases/%s` be moved to a spanner specific class, like the accessor?

Reply:
Replaced it with a constant.